### PR TITLE
pylhc submitter pages

### LINKED ISSUE #43: Add pylhc_submitter to OMC packages
update job submitter entry in github omc wiki

### PR DESCRIPTION
**Pull request by mihofer**
Add a separate category and pages for the new pylhc-submitter.
Fixes #43  and #44

### CODE REVIEW

File: docs/packages/pylhcsubmitter/autosix.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    
+    - **ssh** *(str)*: Run htcondor from this machine via ssh (needs access to the
+        `working_directory`)
+    
+    - **stop_workspace_init**: Stops the workspace creation before initialization, so one can make
+        manual changes.
+        action: ``store_true``
+    
+    - **unlock**: Forces unlocking of folders.
+        action: ``store_true``
```
Comment (JoschD) [inline docs/packages/pylhcsubmitter/autosix.md:72]: I would just link to the other doc. If you copy paste here we will have to update always twice and I am very certain that already the first update on either end will forget the other one

Comment (fsoubelet) [reply docs/packages/pylhcsubmitter/autosix.md:72]: I don't mind updating the other one and having an admonition linking to that.

**Comment by fsoubelet** [comment]
Previous commit removes the description + API details parts of the files and links to the (updated after merging https://github.com/pylhc/submitter/pull/11) API doc pages for this.

These refer to the new content of this PR for guides & more info.

File: docs/packages/pylhcsubmitter/autosix.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Because the stages after `submit_mask` and `submit_sixtrack` need only be ran after the jobs on `HTCondor` are completed, they will interrupt execution if previous stages have successfully finished.
+To have `AutoSix` continue its work, check your scheduler via `condor_q` and run your script again after everything is done.
+It will pick up where it left as written in the **stages\_completed.txt** file.
+
+!!! tip
+    While most studies should be fine with the input options given, there is the possibility to manually adapt the **sixdeskenv** and **sysenv** files before workspace initialization by using the `stop_workspace_init` flag.
+    This will not reset automatically and one will have to remove the switch again to continue.
+    As this is a bit of a workaround, it might be easier to define a new variable in the **pylhc_submitter.sixdesk_tools.mask_sixdeskenv** file and put its current default into **pylhc_submitter.constants.autosix.SIXENV_DEFAULT**.
+    One can then use the `replace_dict` to change its value.
+    This is left open as a task for the inspired user to implement, as only he knows which variable he or she needs to change.
```
Comment (fsoubelet) [inline docs/packages/pylhcsubmitter/autosix.md:108]: I have left this segment untouched because I'm not sure I get what this is trying to tell me. @JoschD

Comment (JoschD) [reply docs/packages/pylhcsubmitter/autosix.md:108]: You have to have been there to understand...

**Review by JoschD — DISMISSED** [review]

**Review by fsoubelet — APPROVED** [review]

**Pull request merged**